MonoHbb/MonoHbbAllSamplesReader.py

```diff
@@ -1,14 +1,22 @@
 import os 
 
-f = open('allfiles.txt','r')
-outdir = "bbMETSamples_all_lim"
+parallel=False
 
-os.system("mkdir -p "+outdir+"/log")
+f = open('allfiles4.txt','r')
+outdir = "bbMETSamples_large"
+
+if parallel:
+    os.system("mkdir -p "+outdir+"/log")
 
 for ifile in f: 
     filename =  ifile.rstrip()
     print "\n=======================\nSubmitting",filename,"\n=======================\n"
-    #os.system('python MonoHBranchReader.py -i '+filename+' -a')
-    command='nohup python MonoHBranchReader.py -a -i '+filename+' -D '+outdir+' &> '+outdir+'/log/'+filename.split('/')[-1].split('.')[0]+'.log &'
+    
+    if parallel:
+        command='nohup python MonoHBranchReader.py -a -i '+filename+' -D '+outdir+' &> '+outdir+'/log/'+filename.split('/')[-1].split('.')[0]+'.log &'
+    else:
```

Comment:
this case is for hadding later?

Reply:
This submits local jobs parallelly instead of sequentially, that's all.

Reply:
okay